web/app/models/alert.py

```diff
@@ -34,12 +34,14 @@ class AlertView(RowActionListMixin, MyModelView):
 
     list_template = 'import-admin-list-to-dashboard.html'
 
-    # column_list = ["id", "name", "company_name", "created_at"]
+    column_list = ["name", "from_email", "to_addresses", "subject", "body", "html_body", "alert_status"]
     column_searchable_list = ["name", "from_email", "to_addresses"]
     # form_widget_args = {
     #     "name": {"readonly": True},
     # }
```

Comment:
dont leave old commented code - simply remove it, Git remembers all changes

Reply:
I leave it to not scroll documentation all the time. Still need it for 1 feature. I will clean it by separate PR, ok?